### ISSUE TITLE
Incorrect Websocket client URL in the front end code

### ISSUE THREAD

**Issue by gplanchat** [orocommerce/orocommerce#5]
There is an error on the websocket configuration parameters when not listening to the default HTTP port. The URL built has 2 times the port prefix, such as :

![image](https://cloud.githubusercontent.com/assets/152367/9625308/60f6acaa-5155-11e5-8445-a86c543dab65.png)

My cache is up to date and therefore may not interact.

There is my parameters.yml :

```
# This file is auto-generated during the composer install
parameters:
    database_driver: pdo_mysql
    database_host: 127.0.0.1
    database_port: null
    database_name: oro_commerce
    database_user: oro_commerce
    database_password: password
    mailer_transport: mail
    mailer_host: 127.0.0.1
    mailer_port: null
    mailer_encryption: null
    mailer_user: null
    mailer_password: null
    websocket_address: 0.0.0.0
    websocket_port: 8000
    websocket_bind_address: 0.0.0.0
    websocket_frontend_host: '*'
    websocket_frontend_port: 8000
    websocket_backend_host: '*'
    websocket_backend_port: 8000
    backend_prefix: /admin
    session_handler: session.handler.native_file
    locale: en
    secret: *secret*
    installed: '2015-08-19T14:02:14+02:00'
    assets_version: W3xR8CKOOnLDtKM9gBQ=
```

Maybe I'm misusing the parameters.yml file's parameters values.

**Comment by gplanchat** [comment]
Important note : This issue appeared after the first cache refresh, between the install and this first refresh, all worked well.

**Comment by mbessolov** [comment]
Resolved in d363307eb14c6f9954c79c1053410020fececc02
